v2/mets.xsd

```diff
@@ -17,7 +17,7 @@
       Aaron Elkiss (HathiTrust / University of Michigan)
       Inge Hofsink (KB National Library of the Netherlands)
       Juha Lehtonen (CSC – IT Center for Science, Finland)
-      Andreas Nef (Docuteam GmbH, Baden, Switzerland)
+      Andreas Nef (On the record AB, Sweden)
       Tobias Steinke (German National Library)
       Robin Wendler (Harvard University)
 
```

Comment:
We will also have to cleanup the versions and dates here once we're good for the final version.